src/app/components/media/MediaLanguageChip.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay/compat';
+import PropTypes from 'prop-types';
+import Relay from 'react-relay/classic';
+import { FormattedMessage } from 'react-intl';
+import Chip from '@material-ui/core/Chip';
+import EditIcon from '@material-ui/icons/Edit';
+import LanguageIcon from '@material-ui/icons/Language';
+import { can } from '../Can';
+import { withSetFlashMessage } from '../FlashMessage';
+import LanguagePickerDialog from '../layout/LanguagePickerDialog';
+import { getErrorMessage } from '../../helpers';
+import { stringHelper } from '../../customHelpers';
+import UpdateLanguageMutation from '../../relay/mutations/UpdateLanguageMutation';
+
+const MediaLanguageChip = ({ projectMedia, setFlashMessage }) => {
+  const [correctingLanguage, setCorrectingLanguage] = React.useState(false);
+  const [isSaving, setIsSaving] = React.useState(false);
+
+  // FIXME Modernize mutation
```

Comment:
OK, but was there any blocker?

Reply:
Just time. I reused the original mutation. Didn't get to try modernizing it